apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-button/styles.scss

```diff
@@ -1,21 +0,0 @@
-// Added to body to allow general overrides
-.editor-gutenberg-launch__fse-overrides {
-	// Override 'Save' button to have tertiary styles.
-	.edit-post-header__settings {
-		.editor-post-publish-button__button {
-			color: #007cba !important;
-			background: none !important;
-			border: none !important;
-			box-shadow: none !important;
-
-			&[aria-disabled="true"] {
-				opacity: 0.3 !important;
-			}
-		}
-	}
-}
-
-.editor-gutenberg-launch__launch-button {
```

Comment:
We can also likely delete this then:

https://github.com/Automattic/wp-calypso/blob/b7a4a072ae2a13785fb6ccbf9c34ed8e65739705/packages/launch/src/focused-launch/style.scss#L7-L9